RELEASE/scripts/sl_ascend.ash

```diff
@@ -12595,13 +12636,23 @@ boolean L9_oilPeak()
 			}
 		}
 	}
-	addToMaximize("1000ml 100max");
+
+
+	print("Oil Peak with ML: " + monster_level_adjustment(), "blue");
+
 	slAdv(1, $location[Oil Peak]);
-	if(get_property("lastAdventure") == "Unimpressed with Pressure")
-	{
-		set_property("oilPeakProgress", 0.0);
-	}
+
+// This does nothing, get_property("lastAdventure") only returns "Oil Peak" at this point
+//	if(get_property("lastAdventure") == "Unimpressed with Pressure")
```

Comment:
I think "lastEncounter" should work?

Reply:
I just did a little test and it looks like it does. Awesome! Thank you.

Reply:
Fixed.